kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/DefaultConfiguration.java

```diff
@@ -132,7 +132,7 @@ public static DefaultConfiguration fromMap(Map<String, String> map) {
                     asUrlOrResource(getStringProperty(ENVIRONMENT_TEARDOWN_SCRIPT_URL, map, null)))
                 .withEnvironmentConfigUrl(getKubernetesConfigurationUrl(map))
                 .withEnvironmentDependencies(
-                    asURL(Strings.splitAndTrimAsList(getStringProperty(ENVIRONMENT_DEPENDENCIES, map, ""), "\\s+")))
+                    asURL(Strings.splitAndTrimAsList(getStringProperty(ENVIRONMENT_DEPENDENCIES, map, ""), ",\\s*")))
```

Comment:
How about `\\s*,\\s*` regex?

Reply:
I don't mind, let me update it :)

Reply:
I am not sure about any of both expressions, I think that the comma part should be optional, so the property can be either `mykubernetes.json` or `mykubernetes.json, mysecondkubernetes.json`

Reply:
With `\\s*,\\s*` regex following cases will be covered:
* mykubernetes.json 
* mykubernetes.json, mysecondkubernetes.json
* mykubernetes.json , mysecondkubernetes.json

Reply:
Well, you're using `String.split` java method for separating multiple values by regex, so if there's just one single value, nothing happens and this value will be returned as is.